pkg/cli/cli.go

```diff
@@ -153,6 +192,40 @@ func (c *cli) initialize() error {
 		return err
 	}
 
+	// With or without a config, a user can:
+	// 1. Not set --plugins
+	// 2. Set --plugins to a plugin, ex. --plugins=go
+	// In case 1, either default plugins or the config layout can be used to
+	// determine which plugin to use. Otherwise, the value passed to --plugins
+	// is used. Since both layout and a --plugins value are user input, they
+	// may need to be resolved to known plugin keys.
+	var filterKeys map[string]string
+	switch {
+	case len(c.cliPluginKeys) != 0:
+		// Filter plugins by keys passed in CLI.
+		filterKeys = c.cliPluginKeys
+	case c.configured && !projectConfig.IsV1():
```

Comment:
doesn't this break current `v2` projects?

Reply:
It will, and probably a good reason to upgrade to project version "3-alpha".